src/main.rs

```diff
@@ -51,14 +50,7 @@ fn calculate_safety_status(
     has_carrot: bool,
     friends_nearby: i32,
 ) -> bool {
-    if (!wolves_nearby && day_time) || has_carrot {
-        true
-    } else if (!wolves_nearby && day_time) || (friends_nearby > 3) {
-        true
-    } else {
-        false
-    }
-}
+    if (!wolves_nearby && day_time) || has_carrot || (friends_nearby > 3){true} else {false}}
```

Comment:
Good stuff, but we don't actually need the if statement there at all :) that expression that the if statement is checking (`(!wolves_nearby && day_time) || has_carrot || (friends_nearby > 3)`) will actually evaluate to a boolean on its own, so you can just return that :)

Reply:
Oh I see, Thank you